packages/utilities/src/setFocusVisibility.ts

```diff
@@ -14,22 +15,25 @@ export const IsFocusHiddenClassName = 'ms-Fabric--isFocusHidden';
  *
  * @param enabled - Whether to turn focus visibility on or off.
  * @param target - Optional target from which to get window in case no `providerElem` has been specified.
- * @param providerElem - Theme provider element to which the focus visibility classnames are attached. If no provider
- *                       element is passed in, the classnames are attached to the document body that contains `target`.
+ * @param registeredProviders - Array of provider refs that are associated with a FocusRectsProvider. If no array
+ *                              is passed in, the classnames are attached to the document body that contains `target`.
  */
-export function setFocusVisibility(enabled: boolean, target?: Element, providerElem?: Element): void {
-  let classList;
-  if (providerElem) {
-    classList = providerElem.classList;
+export function setFocusVisibility(
+  enabled: boolean,
+  target?: Element,
+  registeredProviders?: React.RefObject<HTMLElement>[],
+): void {
+  const updateClassList = (el: HTMLElement) => {
+    el.classList.add(enabled ? IsFocusVisibleClassName : IsFocusHiddenClassName);
+    el.classList.remove(enabled ? IsFocusHiddenClassName : IsFocusVisibleClassName);
+  };
+
+  if (registeredProviders) {
+    registeredProviders.forEach(ref => ref.current && updateClassList(ref.current));
   } else {
     const win = target ? getWindow(target) : getWindow();
```

Comment:
Super nit: `getWindow()` already handles undefined and null so this could be reduced to

```suggestion
    const win = getWindow(target);
```